src/coresyf_image_crop/coresyf_image_crop.py

```diff
@@ -76,6 +77,40 @@ def get_shapefile_crs(data_source):
     return epsg_code
 
 
+def crop_raster(input_raster, output_raster_file, polygon_extent, dest_crs):
+    '''
+    Crops the image specified by input_raster using the limits defined in
+    polygon_extent (buffer already applied).
```

Comment:
Remove "(buffer already applied)". It is not important for the function description if the polygon has buffer already applied or not. It is just a suggestion that helps to keep the function description generic.